aunt-leahs-app/src/components/select-box/select-box.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
```

Comment:
You should be able to use destructuring for your import statements. I think the following should work:

import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';